packages/design-system/src/components/Banner/Banner.test.js

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import CloseIcon from 'react-material-icon-svg/dist/CloseIcon';
+
+import Banner from './Banner';
+
+describe('Banner', () => {
+  it('should render small info Banner', () => {
+    const component = shallow(<Banner text="Example text" />);
+
+    expect(component).toMatchSnapshot();
+  });
+
+  it('should render large Banner', () => {
```

Comment:
I think we agreed on one of the weekly dev meetings that tests "it should render" are not helpful at all. Also, you could write it smarter by using for loop and not repeating the code.

Reply:
yes we agreed. in agent app. imho design system is a good place for snapshots.